### PR TITLE
Update Spacy to 3.6.1

### PR DESCRIPTION
**Pull request by jomonson**

### CODE REVIEW

File: requirements.txt [hunk 1]
```diff
@@ -8,7 +8,7 @@ julius
 num2words
 numpy
 sentencepiece
-spacy==3.5.2
+spacy==3.6.1
```
Comment (moejay) [inline requirements.txt:11]: I want this to, but to keep backwards compat, is it possible to keep it at spacy>=3.5.2?

Comment (jomonson) [reply requirements.txt]: Sure, changed to the soft limit. Also looked at the Spacy upgrading guide [here](https://spacy.io/usage/v3-6#upgrading) and it doesn't look like there should be a problem. Current code doesn't use Spacy cfg files or `Span` objects.

**Comment by jomonson** [comment]
@moejay if there are no additional requirements, can we push forward to run workflows?

**Comment by moejay** [comment]
@jomonson I'm not a maintainer :) I can't do that

**Review by moejay — APPROVED** [review]

**Comment by JadeCopet** [comment]
Solved in #355 which was merged earlier.

**Pull request closed without merging**